frontend/packages/employee-frontend/src/components/unit/unit-details/UnitEditor.tsx

```diff
@@ -437,9 +442,9 @@ function toFormData(unit: Unit | undefined): FormData {
       : type?.includes('CENTRE')
       ? 'CENTRE'
       : undefined,
-    canApplyDaycare: unit?.canApplyDaycare ?? false,
-    canApplyPreschool: unit?.canApplyPreschool ?? false,
-    canApplyClub: unit?.canApplyClub ?? false,
+    daycareApplyPeriod: unit?.daycareApplyPeriod ?? null,
```

Comment:
Do we really need the default values here? "if value is null, return null instead"
There more of this later too...IMHO it makes the code less readable since it adds a pointless "branch" you need to think about when trying to understand the code

Reply:
hmm I think the issue is that the value may be undefined and that needs to be defaulted to null instead

Reply:
Hmm I think I picked a bad example...I think there was another place where there was no mixing of undefined and null.

Of course this raises another question: why do we mix undefined and null in the first place? (this is a bigger issue and not specific to this pull request)

Reply:
Removed those other cases I could find. Yeah our undefined/null usage would need general cleanup.